shared/ui-components/src/page_components/listing/AdditionalFees.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react"
 import { Listing } from "@bloom-housing/core"
-import t from "@bloom-housing/ui-components/src/helpers/translator"
+import { t } from "@bloom-housing/ui-components/src/helpers/translator"
```

Comment:
Another relative import (we're inside `ui-components`):

```suggestion
import { t } from "../../helpers/translator"
```